constellation_base/models.py

```diff
@@ -39,3 +42,20 @@ def settings_dict(self):
             'organization': self.organization,
             'titlebar_links': self.titlebar_links,
         }
+
+
+class LocalGroupACL(models.Model):
+    """ACL for Local Groups that don't exist in other authentication sources"""
+    group = models.ForeignKey(Group)
+
+    # This contains a delimited string of usernames that are members of this
+    # group.  The delimiter is not important because the check is just done
+    # with a 'user in groupACL.members' style check.
+    members = models.TextField()
```

Comment:
A delimited field bothers me.  A more correct way to do this would be to mimic the `auth_user_groups` table, mapping user_id's to group_id's (via many-to-many or something).